### ISSUE TITLE
Right clicking a document tab crashes Visual Studio for Mac 8.1

### ISSUE THREAD

**Issue by mrward** [DarkCloud14/MonoDevelop.StyleCop#26]
There have been some API changes in Visual Studio for Mac 8.1 which seem to be causing the StyleCop addin to crash the IDE when you right click a document tab.

StyleCop Support 1.1.0.1

```
	  at <unknown> <0xffffffff>
	  at MonoDevelop.StyleCop.BaseAnalysisHandler:Update <0x00119>
	  at MonoDevelop.StyleCop.ActiveDocumentAnalysisHandler:Update <0x0006a>
	  at MonoDevelop.Components.Commands.CommandHandler:InternalUpdate <0x00015>
	  at MonoDevelop.Components.Commands.CommandManager:DefaultUpdateCommandInfo <0x00262>
	  at MonoDevelop.Components.Commands.CommandManager:GetCommandInfo <0x00742>
	  at MonoDevelop.Components.Commands.CommandManager:GetCommandInfo <0x0003a>
	  at MonoDevelop.Components.Mac.MDMenuItem:Update <0x000aa>
	  at MonoDevelop.Components.Mac.MDMenu:UpdateCommands <0x00125>
	  at MonoDevelop.Components.Mac.MDMenu:MenuNeedsUpdate <0x000b2>
	  at <Module>:runtime_invoke_void__this___object <0x000e6>
	  at <unknown> <0xffffffff>
	  at ObjCRuntime.Messaging:void_objc_msgSend_IntPtr_IntPtr_IntPtr <0x0007b>
	  at AppKit.NSMenu:PopUpContextMenu <0x000b2>
	  at <>c__DisplayClass4_0:<ShowContextMenu>b__0 <0x00402>
	  at InvokeProxy:HandlerInternal <0x0008d>
	  at InvokeProxy:HandlerInternal <0x00062>
	  at <unknown> <0xffffffff>
	  at Gtk.Application:gtk_main <0x00056>
	  at Gtk.Application:Run <0x00012>
	  at MonoDevelop.Ide.IdeStartup:Run <0x011ea>
	  at MonoDevelop.Ide.IdeStartup:Main <0x002ca>
	  at Xamarin.Startup.MainClass:Main <0x0001a>
	  at <Module>:runtime_invoke_int_object <0x0010c>
```

**Comment by DarkCloud14** [comment]
Thanks for the report, will have a look at it, looks like there were additional changes too as at least for me even running StyleCop using the other options doesn't seem to do anything..

Trying to get this fixed asap.

**Comment by DarkCloud14** [comment]
Problem is fixed in latest v1.1.0.2 which should be downloadable using the extension manager of VS for Mac (a refresh might be necessary).
